services/thermal_mgr/thermal_mgr.h

```diff
@@ -5,6 +5,7 @@
 
 typedef enum {
   THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
+  EVENT_INTERRUPT,
```

Comment:
nit: try to keep enum naming consistent by naming this something like `THERMAL_MGR_EVENT_INTERRUPT` so that everything starts with `THERMAL_MGR_EVENT` especially since this is in a .h file and `EVENT_INTERRUPT` is very general